server/server.js

```diff
@@ -1,8 +1,26 @@
 var express = require('express');
 var app = express();
-var port = 1993;
+var port = 3000;
+var Location = require('../db/db.js');
 
 app.use(express.static(__dirname + '/../client/dist'));
 
 
+// where does app.get grab Location ID from? Currently in my db, it's 1-10
+
+app.get(`/locations/:locationID/reviews`, function(req, res) {
+  //this should now readAll data from the database
+  // therefore import mongoose function that will go and read the data in the database
+  console.log('req.params.locationID: ', req.params.locationID)
+  Location.findOne({locationID: parseInt(req.params.locationID)}, function(err, result) {
+    if (err) {
+      console.log(err)

```

Comment:
@harrisonywu @krise3 The server should send an appropriate response to the client in the event of an error (even if it's only a status code 500)